pagination/keysetpagination/paginator.go

```diff
@@ -127,6 +210,16 @@ func WithSize(size int) Option {
 	}
 }
 
+func WithColumn(name string, order Order) Option {
```

Comment:
Do you plan on exposing the order as a parameter to the HTTP client? In that case I think it would be better to not make it part of this option, but rather make it part of the token.
I mean, the token is basically a pointer to a row in a table of certain order. Therefore the order should also be derivable from the token correct?

Reply:
Hm, in general it might not make sense to have this as an option. What happens when I first make a request to `/identities?page_size=100&additional_column=created_at,DESC`, but then the subsequent request is `/identities?page_size=100&page_token=.......`?

Reply:
No, not yet. I think we need a more sophisticated method of transporting that information from the client to the library, than just delimited strings. I have some ideas, but I need to finish the courier lists first. After that, we can have another hack at this.

Reply:
We just have to mind breaking changes, and should not redo it in a different way later on...